### PR TITLE
Develop/stats page

### PR DESCRIPTION
**Pull request by mzschunke**
[US6](https://github.com/mzschunke/capstone-project/issues/15)

Remark for the drop down selection: 
ATM there are a lot of (older) players w/o any data. In order to test the displayed tables, please select an active NBA player ;) Here are some examples: Giannis Antetounkoumpo, LeBron James, Anthony Davis, Jayson Tatum...

### CODE REVIEW

File: pages/stats/index.js
Comment (doctype-Melvin) [inline pages/stats/index.js:1]: Great work Mo. I see your app comes together nicely. 
Something I've noticed is that you use a styled component for the StatsContainer. 
You could consider making your Stats component more concise by creating a focused component for the player stats rendering.

You've got this!

**Review by doctype-Melvin — COMMENTED** [review]
Moritz, you're doing great.
Thinking about taking some of your code and using it in my app, too!
I just left a comment about making your code a bit more maintainable and concise. 
Keep up the good work!

File: pages/stats/index.js
Comment (mzschunke) [reply pages/stats/index.js:1]: Hi Melvin! First of all: Thanks heaps for taking the time to look into my code and your encouraging comments! Makes me somewhat proud if you consider using parts of my code for your app! =)

I am using the "StatsContainer" as a styled component to have the table being rendered in the center of the stats page. I'm not quite sure what you mean by "focused component" - could you kindly explain a little further or link an example? :) Cheers!

File: pages/stats/index.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            <TD>{playerStats.ast}</TD>
+            <TD>{playerStats.reb}</TD>
+            <TD>{playerStats.stl}</TD>
+            <TD>{playerStats.blk}</TD>
+            <TD>{playerStats.turnover}</TD>
+            <TD>{playerStats.fg_pct}</TD>
+            <TD>{playerStats.fg3_pct}</TD>
+            </tbody>
+          </StyledTable>
+          </StatsContainer>
```
Comment (doctype-Melvin) [inline pages/stats/index.js:125]: You could outsource this part into its own component and pass it the playerStats. It's just a suggestion. Let me know what you think.

**Review by doctype-Melvin — COMMENTED** [review]
Sorry, I saw your reply just now. I've highlighted the parts which I think could also make up a separate component. Again, this is just a suggestion for brevity's sake and I'm not requesting that you change this.

File: pages/stats/index.js (hunk 1 above)
Comment (mzschunke) [reply pages/stats/index.js]: Now I got you.

Thanks for the suggestion - I happily implemented it. :)

Comment (doctype-Melvin) [reply pages/stats/index.js]: Let's gooo! xD

**Pull request merged**